### PR TITLE
Sidney1120 branch test

### LINKED ISSUE #4787: sidney1120_add me now
Hello! :wave: Please add me as a collaborator to this repo. 
test

### PR DESCRIPTION
**Pull request by sidney1120**
adding test branch
test test test

closes #4787

### CODE REVIEW

**Comment by sidney1120** [comment]
test

**Comment by sidney1120** [comment]
test

**Comment by sidney1120** [comment]
test not work

**Comment by LinghengMeng** [comment]
👍

File: _pins/sidney1120 [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+githubHandle: sidney1120
+latitude: 37.77
+longitude: -122.345
```
Comment (hectorsector) [inline _pins/sidney1120]: @sidney1120 
Great work so far! You'll need to make sure that your code matches exactly the template in the class, including `---` before and after the code:
```
---
githubHandle: YOUR-USERNAME
latitude: YOUR_LATITUDE
longitude: YOUR_LONGITUDE
---
```

Comment (hectorsector) [inline _pins/sidney1120]: You'll also need to rename your file: `_pins/sydney1120.json` and remove the file `_pins/AAA.sidney`. Please let us know if we can help you do that!

**Comment by hollenberry** [comment]
Hey @sidney1120  ! As noted by Hector above, if you make those requested changes, you'll be set. 👍

Please let us know if you need any help! We'll be closing this out to keep the repository clean if there's no action or conversation within the week.

**Comment by crichID** [comment]
@sidney1120 I went ahead and fixed the syntax issues in https://github.com/githubschool/open-enrollment-classes-introduction-to-github/pull/4793/commits/5f7d93133532daa491d0d0501e75852e048b129d. Please take a quick look at the changes. Once the checks have finished, please feel free to click the big green **Merge pull request** button. If you haven't merged it by April 21st, I will go ahead and do it just to keep the repository clean!

**Pull request merged**